library/src/main/java/com/novoda/downloadmanager/lib/DownloadProvider.java

```diff
@@ -496,6 +307,12 @@ public String getType(final Uri uri) {
                     return mimeType;
                 }
             }
+            case BATCHES: {
```

Comment:
if you battle to have these in our standards then I'll battle for squiggles on new lines 

```java
public Cant handleMy()
{
  return swag();
}
```